### PR TITLE
feat: bangla language support

### PR DESCRIPTION
**Pull request by hmasum52**
### Features
- [x] Multi language support
- [x] Translations for balance screen
- [x] Translations for fare calculation screen

### Preview
image

### CODE REVIEW

File: composeApp/src/commonMain/composeResources/values-bn/strings.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<resources>
+    <string name="appName">এমআরটি বাডি</string>
```
Comment (aniruddha-adhikary) [inline composeApp/src/commonMain/composeResources/values-bn/strings.xml:2]: Should the name remain in English?

File: composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/App.kt [hunk 2]
```diff
@@ -45,11 +58,54 @@ fun App(dao: DemoDao) {
...
+        var lang by remember { mutableStateOf(Language.English.isoFormat) }
+        LocalizedApp (
+            language= lang
+        ) {
+            Scaffold {
+                Box(
+                   Modifier.systemBarsPadding()
+                ) {
+                    Column {
+                        TopAppBar(
```
Comment (aniruddha-adhikary) [inline composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/App.kt:72]: This eats quite a bit of space from my small screen. Could we possibly move this to a Settings Screen? (We can land these changes in a feature branch and work from there I suppose?)

File: composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/App.kt [hunk 3]
```diff
@@ -45,11 +58,54 @@ fun App(dao: DemoDao) {
...
+
+
+        }
+
+
+    }
+}
+
+private fun switchLanguage(lang: String) : String{
+    print("Switching language")
```
Comment (aniruddha-adhikary) [inline composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/App.kt:105]: Dangling print? :D

File: composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/App.kt [hunk 4]
```diff
@@ -45,11 +58,54 @@ fun App(dao: DemoDao) {
...
+                            backgroundColor = MaterialTheme.colors.background,
+                            actions = {
+                                OutlinedButton(
+                                    onClick = {
+                                        lang = switchLanguage(lang)
+                                        changeLang(lang)
+                                    },
+                                    shape = RoundedCornerShape(24.dp),
+                                ) {
+                                    Text(text = stringResource(Res.string.language))
```
Comment (aniruddha-adhikary) [inline composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/App.kt:85]: Perhaps the text should be the name of the Other language if we are in a language toggle situation. Like the guy who doesn't know English cannot read the word English (ok very hypothetical and)

File: composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/nfc/service/TimestampService.kt [hunk 5]
```diff
@@ -24,7 +70,7 @@ class TimestampService {
         val validDay = if (day in 1..31) day else 1
 
         // Create a LocalDateTime instance
-        val dateTime = LocalDateTime(
+        return LocalDateTime(
```
Comment (aniruddha-adhikary) [inline composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/nfc/service/TimestampService.kt:73]: THANK YOU!

File: composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/nfc/service/TimestampService.kt [hunk 6]
```diff
@@ -1,16 +1,62 @@
...
+            val day = translateNumber(dateTime.dayOfMonth).padStart(2, zero)
+            val month = getMonth(dateTime.monthNumber)
+            val year = translateNumber(dateTime.year)
+
+            return "$day $month $year, ${getHour(dateTime.hour)}:$zero$zero ${getAmPm(dateTime.hour)}"
+        }
+
+        fun getAmPm(hour: Int): String {
+            if (Locale.current.language == "bn") {
+                return if (hour < 12) "এএম" else "পিএম"
```
Comment (aniruddha-adhikary) [inline composeApp/src/commonMain/kotlin/net/adhikary/mrtbuddy/nfc/service/TimestampService.kt:24]: idk how to feel about this.

File: composeApp/src/commonMain/composeResources/values-bn/strings.xml (hunk 1 above)
Comment (kazimmt) [reply composeApp/src/commonMain/composeResources/values-bn/strings.xml:2]: > Should the name remain in English?

YES

**Pull request merged**